src/test/scala/chiselTests/experimental/hierarchy/InstanceSpec.scala

```diff
@@ -413,6 +413,33 @@ class InstanceSpec extends ChiselFunSpec with Utils {
       val (chirrtl, _) = getFirrtlAndAnnos(new AddTwoNestedInstantiableDataWrapper(4))
       exactly(3, chirrtl.serialize.split('\n')) should include("i1.in <= i0.out")
     }
+    it("(3.r): @probe should create probes") {
```

Comment:
probe IOs right? so the idea is the `@probe` macro is equivalent to 

- create the probe IO
- create the probed-value
- define the probe IO to the probed-value?

is any @probe automatically @public, which lookup of returns the handle to the probe IO?